tests/base.py

```diff
@@ -263,11 +263,10 @@ def modify_expected_record(self, expected_record):
                 else:
                     self.align_date_type(expected_record, key, value)
 
-
     def align_date_type(self, record, key, value):
         """datetime values must conform to ISO-8601 or they will be rejected by the gate"""
-        # TODO update this to execute fo all datetime objects
-        if isinstance(value, str) and key in ['updated_at', 'created_at']:# TODO: update to reflect replication keys
+        if isinstance(value, str) and isinstance(self.parse_date(value), dt):
```

Comment:
Can we use type hints to avoid at least the first isinstance check?

Reply:
Well everything is coming through this. So adding type hints would cause the method to fail. We could add them and then do the type checking in the other method. But that would just move the `isinstance`